app/helpers/data_sharing_agreement_helper.rb

```diff
@@ -61,7 +61,11 @@ def present_section_12_2(data_protection_confirmation:, user:, organisation:)
       end
     end
 
-    "12.2. For #{@org_name}: Name: #{@dpo_name}, Postal Address: #{@org_address}, E-mail address: #{@dpo_email}, Telephone number: #{@org_phone}"
+    if data_protection_confirmation&.confirmed? && @dpo_email.exclude?("@") # Do not show invalid email addresses
+      "12.2. For #{@org_name}: Name: #{@dpo_name}, Postal Address: #{@org_address}, Telephone number: #{@org_phone}"
+    else
+      "12.2. For #{@org_name}: Name: #{@dpo_name}, Postal Address: #{@org_address}, E-mail address: #{@dpo_email}, Telephone number: #{@org_phone}"
+    end
```

Comment:
I'd flip this and use `.include?` instead, but that's just preference
Would it drop into `else` if `data_protection_confirmation` is not confirmed?

Reply:
Sure!